submodules/k8s/variables.tf

```diff
@@ -22,6 +22,11 @@ variable "bastion_public_ip" {
   description = "Bastion host public ip."
 }
 
+variable "eks_cluster_arn" {
```

Comment:
we could just ask for the server name.

Reply:
The only ways I could see to get it (without resorting to awscli, which I assume isn't the suggestion) bring us back to text parsing of some sort, since the config options don't have any sort of `jsonpath`-type output options:

```
>kubectl --kubeconfig ./kubeconfig config get-contexts | egrep "^\*" | awk '{ print $3 }'
arn:aws:eks:us-west-2:0123456789:cluster/chibipug6680
>kubectl --kubeconfig ./kubeconfig config get-clusters | egrep -v NAME | head -1
arn:aws:eks:us-west-2:0123456789:cluster/chibipug6680
```